velox/functions/prestosql/DecimalFunctions.cpp

```diff
@@ -364,7 +364,8 @@ void registerDecimalMultiply(const std::string& prefix) {
       exec::SignatureVariable(
           S3::name(),
           fmt::format(
-              "{a_scale} + {b_scale}",
+              "{a_scale} + {b_scale}", /*When its value exceeds precision,
```

Comment:
perhaps, add a comment above the code

```
// Result type resolution fails if sum of input scales exceeds 38.
"{a_scale} + {b_scale}",
```

I suggest to change the comment to make it more accurate. Velox doesn't resolve return types outside of tests. It is an application's job to do so.

Reply:
Updated, thanks!